flit_core/flit_core/common.py

```diff
@@ -132,15 +132,18 @@ def get_docstring_and_version_via_ast(target):
     for child in node.body:
         # Only use the version from the given module if it's a simple
         # string assignment to __version__
-        is_version_str = (
-                isinstance(child, ast.Assign)
-                and len(child.targets) == 1
-                and isinstance(child.targets[0], ast.Name)
-                and child.targets[0].id == "__version__"
-                and isinstance(child.value, ast.Str)
-        )
-        if is_version_str:
-            version = child.value.s
+        if isinstance(child, ast.Assign):
+            for target in child.targets:
+                is_version_str = (
+                    isinstance(target, ast.Name)
+                    and target.id == "__version__"
+                    and isinstance(child.value, ast.Str)
+                )
+                if is_version_str:
+                    version = child.value.s
+                    break
+            else:
+                continue
```

Comment:
I think there's a good opportunity to use the `any()` builtin here and make this clearer.


```suggestion
        is_version_str = (
                isinstance(child, ast.Assign)
                and any(
                    isinstance(target, ast.Name)
                    and target.id == "__version__"
                    for target in child.targets
                )
                and isinstance(child.value, ast.Str)
        )
        if is_version_str:
            version = child.value.s
            break
```